util-taglib/src/META-INF/liferay-ui.tld

```diff
@@ -1304,6 +1304,13 @@
 			<required>false</required>
 			<rtexprvalue>true</rtexprvalue>
 		</attribute>
+		<attribute>
+			<description>Whether the checkbox is selected by default. The default value is <![CDATA[<code>true</code>]]>.</description>
```

Comment:
I think it should be more clarify in here. Maybe it is should "Whether the permissionPropagationCheckbox is selected by default."

Reply:
Fixed it. Thanks